packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -245,21 +258,6 @@ export interface IContainerRuntimeBase extends IEventProvider<IContainerRuntimeB
  * @alpha
  */
 export interface IFluidDataStoreChannel extends IDisposable {
-	readonly id: string;
```

Comment:
We should very carefully study if removal of ID going to break Loop. I'm 99% sure they use ID, though I really hope they do not depend in any way on this interface - only on FLuidDataStoreRuntime class, and maybe IDataStoreRuntime interface.

In general, can you please check (by doing quick code inspection) if any changes in interfaces / contracts are going to break Loop?

Reply:
Sure.

Reply:
This specific instance (for this interface, I mean) we should be fine.

For the rest, I think just bumping loop to a dev version of Fluid should give us more information. I'll try to see if I can get some progress in that direction.

Reply:
FYI - I tagged you in one of the comments in my PR, we will run into collision here, as I use ID in DataStore class